lib/devise_two_factor/spec_helpers/two_factor_authenticatable_shared_examples.rb

```diff
@@ -96,14 +96,15 @@
     let(:otp_secret_length) { subject.class.otp_secret_length }
     let(:account)           { Faker::Internet.email }
     let(:issuer)            { "Tinfoil" }
+    let(:encoded_account)   { ERB::Util.url_encode(account).to_s }
```

Comment:
This is now being used by the new rotp instead of the deprecated `URI.encode` that we get warning spams about.